mappings/jy.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS jy
```

Comment:
This class appears to hold the recipes for the Workbench, Furnace, Chest, and (weirdly) Sandstone
Maybe give it an appropriate name?

Reply:
a placeholder name is better than nothing..
How about `UtilityTileRecipes` for now